.github/workflows/sdk-performance-metrics.yml

```diff
@@ -1,4 +1,4 @@
-name: Metrics
+name: SDK Performance
```

Comment:
That's because on the `Actions` tab it was not possible to determine where was the `Performance` and where was the `Size` metrics.